.github/workflows/ci.yml

```diff
@@ -93,7 +93,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install jaxlib
-        pip install jax
+        pip install jax==0.1.70
```

Comment:
I temporarily restrict JAX version because the example `ode` is failing with the latest version, probably due to [this PR](https://github.com/google/jax/pull/3562). I'll try to make a reproducible code to report it.